test/cmdLineTests/runtimemxbeanTests/getPidTest.pl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+##############################################################################
+#  Copyright (c) 2017, 2017 IBM Corp. and others
+#
+#  This program and the accompanying materials are made available under
+#  the terms of the Eclipse Public License 2.0 which accompanies this
+#  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+#  or the Apache License, Version 2.0 which accompanies this distribution and
+#  is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  This Source Code may also be made available under the following
+#  Secondary Licenses when the conditions for such availability set
+#  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+#  General Public License, version 2 with the GNU Classpath
+#  Exception [1] and GNU General Public License, version 2 with the
+#  OpenJDK Assembly Exception [2].
+#
+#  [1] https://www.gnu.org/software/classpath/license.html
+#  [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+#  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+##############################################################################
+
+use strict;
+use warnings;
+use IPC::Open3;
```

Comment:
1. Is this submodule available in Perl core from the version we requested in README? 
2. Have this test been tested on Win and Z/OS to make sure they also work on these two platform? Reason for asking this is that sometimes new submodule used in perl script may not work well on Win and Z/OS.

Reply:
IPC::Open3 is available in Perl 5.10.1, which we require: http://perldoc.perl.org/5.10.1/IPC/Open3.html